security/advisories/2023-31047/index.html

```diff
@@ -0,0 +1,71 @@
+---
+layout: security-advisory
+filename: security
+title: CVE-2023-31047 Potential bypass of validation
```

Comment:
Can we reasonable get "Django" in here to make it clear it's something upstream?

Reply:
Aligning this with the link in index.html below...